pom.xml

```diff
@@ -16,6 +16,7 @@
     <module>kubernetes</module>
     <module>operator-build-maven-plugin</module>
     <module>json-schema</module>
+    <module>src</module>
```

Comment:
It makes me very uncomfortable to have a module called, "src."  I wonder if it is now time to rethink this directory. It does appear to be a module, but its purpose is not obvious.

Reply:
+1

Reply:
Agreed... I don't remember the history of how we created a top-level directory called "src", but it's time to move it to a more appropriate name. If we use `git mv` then git will preserve the history.

Reply:
It looks like everything in src is related to the operator's packaging, e.g. the operator's own scripts and tests of the same.

Reply:
What do people think about "packaging" or "assembly"?

Reply:
Those names have meanings in the Maven world, so are probably not the best choice.

I'd want to understand, though, why this simply be moved under the operator module? The reasons for that might clarify the name.

Reply:
As we discussed, I'm going to look at moving the files under the operator module.